### PR TITLE
Update signer to capture signing period during repo creation.

### PR DESCRIPTION
**Pull request by kommendorkapten**
```shell
kommendorkapten@m1m14:~/git/tuf-repo-test-1 % playground-delegate fsn-init
Remote branch not found: branching off from main
Creating a new Playground TUF repository

Configuring role root
 1. Configure signers: [@kommendorkapten], requiring 1 signatures
 2. Configure expiry: Role expires in 365 days, re-signing starts 60 days before expiry
Please choose an option or press enter to continue: 2
Please enter root expiry period in days [365]: 56
Please enter root signing period in days [60]: 45
 1. Configure signers: [@kommendorkapten], requiring 1 signatures
 2. Configure expiry: Role expires in 56 days, re-signing starts 45 days before expiry
Please choose an option or press enter to continue:

Configuring role targets
 1. Configure signers: [@kommendorkapten], requiring 1 signatures
 2. Configure expiry: Role expires in 56 days, re-signing starts 45 days before expiry
Please choose an option or press enter to continue:

Configuring online roles
 1. Configure online key: sigstore:
 2. Configure timestamp: Expires in 2 days, re-signing starts 1 days before expiry
 3. Configure snapshot: Expires in 56 days, re-signing starts 45 days before expiry
Please choose an option or press enter to continue: 2
Please enter timestamp expiry in days [2]: 23
Please enter timestamp signing period in days [1]: 11
 1. Configure online key: sigstore:
 2. Configure timestamp: Expires in 23 days, re-signing starts 11 days before expiry
 3. Configure snapshot: Expires in 56 days, re-signing starts 45 days before expiry
Please choose an option or press enter to continue: 3
Please enter snapshot expiry in days [56]: 21
Please enter snapshot signing period in days [45]: 10
 1. Configure online key: sigstore:
 2. Configure timestamp: Expires in 23 days, re-signing starts 11 days before expiry
 3. Configure snapshot: Expires in 21 days, re-signing starts 10 days before expiry
Please choose an option or press enter to continue:
...
```
Yields the following `root.json`:
```json
{
 "signatures": [
  {
   "keyid": "057a658c3f9630254e57444dc6a2f83352b1fab4af69633325a0dd71b6dd8613",
   "sig": "3046022100ecf559fbe279ce51c8db7870255902fe7ad5b08f2141cca0bff431c499c716350221008ab8d6090b88887c30c56e43d0e411df66cb9842b41bfc70ed6445e8773b62e5"
  }
 ],
 "signed": {
  "_type": "root",
  "consistent_snapshot": true,
  "expires": "2023-07-11T08:33:09Z",
  "keys": {
   "057a658c3f9630254e57444dc6a2f83352b1fab4af69633325a0dd71b6dd8613": {
    "keytype": "ecdsa",
    "keyval": {
     "public": "-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE0rNfVtW3SeL8+k+YOr9JrlNUYJz4\nAehc/gALb0xuQnsrb/5yx61wt3mspeqoQgvQ7iXi1uVrq+9GWTjmhHxdpg==\n-----END PUBLIC KEY-----\n"
    },
    "scheme": "ecdsa-sha2-nistp256",
    "x-playground-keyowner": "@kommendorkapten"
   },
   "abcd": {
    "keytype": "sigstore-oidc",
    "keyval": {
     "identity": "https://github.com/kommendorkapten/tuf-repo-test-1/.github/workflows/snapshot.yml@refs/heads/main",
     "issuer": "https://token.actions.githubusercontent.com"
    },
    "scheme": "Fulcio",
    "x-playground-online-uri": "sigstore:"
   },
   "efgh": {
    "keytype": "sigstore-oidc",
    "keyval": {
     "identity": "https://github.com/kommendorkapten/tuf-repo-test-1/.github/workflows/version-bumps.yml@refs/heads/main",
     "issuer": "https://token.actions.githubusercontent.com"
    },
    "scheme": "Fulcio",
    "x-playground-online-uri": "sigstore:"
   }
  },
  "roles": {
   "root": {
    "keyids": [
     "057a658c3f9630254e57444dc6a2f83352b1fab4af69633325a0dd71b6dd8613"
    ],
    "threshold": 1
   },
   "snapshot": {
    "keyids": [
     "abcd",
     "efgh"
    ],
    "threshold": 1,
    "x-playground-expiry-period": 21,
    "x-playground-signing-period": 10
   },
   "targets": {
    "keyids": [
     "057a658c3f9630254e57444dc6a2f83352b1fab4af69633325a0dd71b6dd8613"
    ],
    "threshold": 1
   },
   "timestamp": {
    "keyids": [
     "abcd",
     "efgh"
    ],
    "threshold": 1,
    "x-playground-expiry-period": 23,
    "x-playground-signing-period": 11
   }
  },
  "spec_version": "1.0.31",
  "version": 1,
  "x-playground-expiry-period": 56,
  "x-playground-signing-period": 45
 }
}
```

### CODE REVIEW

File: playground/tests/e2e.sh [hunk 1]
```diff
@@ -28,13 +28,16 @@
 #      + git/ -- the repository used to emulate human user running playground-delegate and sign
 
 set -euo pipefail
-if [ -v DEBUG_TESTS ]; then
+
+DEBUG_TESTS=${DEBUG_TESTS:-}
+
+if [ ! -x ${DEBUG_TESTS} ]; then
```
Comment (jku) [inline playground/tests/e2e.sh]: what does this do? or should it be "-z" like the other instance?

**Review by jku — APPROVED** [review]
LGTM, just one question inline

**Review by jku — APPROVED** [review]
LGTM, just one question inline

File: playground/tests/e2e.sh (hunk 1 above)
Comment (kommendorkapten) [reply playground/tests/e2e.sh]: You managed to look at the next last commit, it's fixed in the last commit. It was a spelling error.

**Pull request merged**